Sources/TuistSigning/ProvisioningProfile/ProvisioningProfile.swift

```diff
@@ -16,9 +16,20 @@ struct ProvisioningProfile: Equatable {
     let applicationIdPrefix: [String]
     let platforms: [String]
     let expirationDate: Date
+
+    struct Content {
```

Comment:
that's a good idea! Maybe `ProvisioningProfile.path` property can be made non-optional now? If not, that's fine, too!